MFVNeutralino/test/jettree.py

```diff
@@ -0,0 +1,56 @@
+from JMTucker.Tools.BasicAnalyzer_cfg import *
+
+is_mc = True # for blinding
+
+from JMTucker.MFVNeutralino.NtupleCommon import ntuple_version_use as version, dataset, use_btag_triggers, use_MET_triggers
+#sample_files(process, 'ttbarht0600_2017' if is_mc else 'JetHT2017B', dataset, 1)
+#sample_files(process, 'ttbarht0600_2017' if is_mc else 'JetHT2017B', dataset, 1)
+#sample_files(process, 'mfv_neu_tau010000um_M0800_2017' if is_mc else 'JetHT2017B', dataset, 10)
+sample_files(process, 'mfv_splitSUSY_tau000001000um_M2000_1800_2017' if is_mc else 'JetHT2017B', dataset, 10)
+#input_files(process,[
+#                    '/uscms/home/ali/nobackup/LLP/CornellCode/mfv_9417/src/JMTucker/MFVNeutralino/test/TestRun/ntuple.root'
+#            ])
+tfileservice(process, 'jettree.root')
+cmssw_from_argv(process)
+
+process.load('JMTucker.MFVNeutralino.WeightProducer_cfi')
+process.load('JMTucker.MFVNeutralino.VertexSelector_cfi')
+process.mfvJetTreer = cms.EDAnalyzer('MFVJetTreer',
+                             mevent_src = cms.InputTag('mfvEvent'),
+                             weight_src = cms.InputTag('mfvWeight'),
+                             tracks_src = cms.InputTag('jmtRescaledTracks'),
+                             #vertextight_src = cms.InputTag('mfvSelectedVerticesTight'),
+                             #vertexloose_src = cms.InputTag('mfvSelectedVerticesTightNtk3or4'),
+                             vertextight_src = cms.InputTag('mfvSelectedVerticesExtraLoose'),
+                             vertexloose_src = cms.InputTag('mfvSelectedVerticesExtraLoose'),
+                             vertexextraloose_src = cms.InputTag('mfvSelectedVerticesExtraLoose'),
```

Comment:
Not sure if these are all intentionally identical?

Reply:
Yeah jettree is the current TTree producer for my ML algorithms, I used to have several different vertex source but not now so I set all of them to be the same now.